yajsapi/executor/index.ts

```diff
@@ -945,8 +964,21 @@ export class Executor {
 
     let payment_client = await this._api_config.payment();
     this._payment_api = new rest.Payment(payment_client);
+
+    let net_client = await this._api_config.net();
+    this._net_api = new rest.Net(net_client);
+
     await stack.enter_async_context(this._wrapped_consumer);
 
+    if (this._network_address) {
+      // TODO: replace with a proper REST API client once ya-client and ya-ts-client are updated
```

Comment:
Let's create a GitHub issue and add it to the SDK project to track this!

Reply:
https://github.com/golemfactory/yajsapi/issues/290